src/containers/BooksForm.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { createBook } from '../redux/actions/index';
+
+export const CATEGORIES = [
+  { name: 'Action', id: '1' },
+  { name: 'Biography', id: '2' },
+  { name: 'History', id: '3' },
+  { name: 'Horror', id: '4' },
+  { name: 'Kids', id: '5' },
+  { name: 'Learning', id: '6' },
+  { name: 'Sci-Fi', id: '7' },
+];
+
+const BooksForm = () => {
+  const dispatch = useDispatch();
+  const [book, setBook] = useState({ title: '', category: '1' });
+
+  const handleChange = (event) => {
+    setBook({ ...book, [event.target.id]: event.target.value });
+  };
+
+  const handleSubmit = () => {
+    dispatch(createBook(book));
+    setBook({ title: '', category: book.category });
+    window.idCounter += 1;
+  };
```

Comment:
- [ ] Add `event.preventDefault();` before dispatch to prevent page refresh after the submit button is clicked.